style.css

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #6070ff;
+  padding: 24px;
+}
+
+.brand {
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.headline {
+  background-image: url("./images/backgroundImage.png");
+  height: calc(100vh - 75px);
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] I have noticed that the background image height is too long :persevere: . Kindly make it professional by setting the reasonable height of the `.headline` :+1: